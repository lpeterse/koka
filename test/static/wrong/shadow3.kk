// shadow parameter
fun id(x,x) = x
