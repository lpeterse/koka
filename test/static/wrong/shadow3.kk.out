test/static/wrong/shadow3.kk(2,10): error: x is already defined at (2, 8)
 hint: if these are potentially recursive definitions, give a full type signature to disambiguate them.